app/controllers/api/v1/pets_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V1::PetsController < ApplicationController
+class Api < V1 < PetsController < ApplicationController
```

Comment:
There's a rule regarding. Let's disable it

Add these two lines in the `rubocop.yml`
```yml
Style/ClassAndModuleChildren:
  Enabled: false
```